guides/common/modules/ref_post-upgrade.adoc

```diff
@@ -0,0 +1,12 @@
+[id="post-upgrade_{context}"]
+= Post Upgrade
```

Comment:
The title is IHMO a bit too generic. Maybe "Post Leapp Update [Steps|Information]"?

Reply:
Those steps do apply for non-leapp upgrades too